src/main/resources/db/changelog/ddl/0018_EIP1-4481_alter_photo_location_arn_column_in_tables.xml

```diff
@@ -0,0 +1,62 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog
+    xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+      http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-4.1.xsd">
+
+    <changeSet author="vishal.gupta@valtech.com" id="0018a_EIP1-4481_add_photo_location_arn_to_certificate_table - add photo_location_arn column" context="ddl">
+        <addColumn tableName="certificate">
+            <column name="photo_location_arn" type="varchar(1024)" afterColumn="status">
+                <constraints nullable="false"/>
+            </column>
+        </addColumn>
+
+        <rollback>
+            <dropColumn tableName="certificate" columnName="photo_location_arn"/>
+        </rollback>
+    </changeSet>
+
+
+    <changeSet author="vishal.gupta@valtech.com" id="0018b_EIP1-4481_certificate_table_data_migration - populate photo_location_arn column of certificate">
+        <sql>
+            <comment>Populating [photo_location_arn] of a certificate entity from any of the matching print_requests [photo_location_arn]</comment>
+            UPDATE certificate cert
+            INNER JOIN print_request pr on cert.id = pr.certificate_id
+            SET cert.photo_location_arn = pr.photo_location_arn
+            WHERE cert.photo_location_arn IS NULL
```

Comment:
As we are going from potentially many print requests to one certificate should we select the latest print request and use its photo location?
Something like:

```
        <sql>
            <comment>Copying photo location</comment>
            UPDATE certificate
            SET photo_location_arn = (SELECT photo_location_arn from print_request pr where certificate.id = pr.certificate_id ORDER BY pr.date_created DESC LIMIT 1)
        </sql>
```

Reply:
haha thanks @Neil-Massey this was my other sql option but we decided to go with this option. @sentioservices @matt-wills-valtech what do you think?

Other option was:

`
UPDATE certificate cert,
            SET cert.photo_location_arn = (SELECT pr2.photo_location_arn from print_request pr2 where pr2.certificate_id = c.id ORDER BY pr2.date_created DESC LIMIT 1)
            WHERE cert.id = pr.certificate_id`

Reply:
_(Still awaiting confirmation around copy data from old column in print_request to new column in certificate. UPDATE query not yet changed as per above Neil's suggestion)_